spring-cloud-azure-appconfiguration-config/src/test/java/com/microsoft/azure/spring/cloud/config/AzureConfigPropertySourceTest.java

```diff
@@ -41,13 +55,14 @@ public static void init() {
         TEST_ITEMS.add(item1);
         TEST_ITEMS.add(item2);
         TEST_ITEMS.add(item3);
+        FEATURE_ITEMS.add(featureItem);
     }
 
     @Before
     public void setup() {
         MockitoAnnotations.initMocks(this);
         propertySource = new AzureConfigPropertySource(TEST_CONTEXT, operations, null, null);
-        when(operations.getKeys(any(), any())).thenReturn(TEST_ITEMS);
+        when(operations.getKeys(any(), any())).thenReturn(TEST_ITEMS).thenReturn(FEATURE_ITEMS);;
```

Comment:
I'm not quite sure here, so we will use `ConfigServiceOperations#getKeys` to access both configuration keys and feature items? Should we use different methods to access these 2 APIs(if there are 2 APIs), prefer to align with .net SDK.

Reply:
There is only 1 API for both. If you look in the portal you will see all feature flags in the Configuration explorer with the rest of the keys.